gvm/protocols/gmpv8.py

```diff
@@ -392,3 +392,153 @@ def modify_credential(
             _xmlkey.add_element("public", public_key)
 
         return self._send_xml_command(cmd)
+
+    def create_tag(
+            self,
+            name,
+            resource_type,
+            *,
+            resource_filter=None,
+            resource_ids=None,
+            value=None,
+            comment=None,
+            active=None
+    ):
+        """Create a tag.
+
+        Arguments:
+            name (str): Name of the tag. A full tag name consisting of
+                namespace and predicate e.g. `foo:bar`
+            resource_type (str): Entity type the tag is to be attached
+                to
+            resource_filter (str, optional) Filter term to select
+                resources the tag is to be attached to.
+            resource_ids (list, optional): IDs of the resources the
+                tag is to be attached to
+            value (str, optional): Value associated with the tag
+            comment (str, optional): Comment for the tag
+            active (boolean, optional): Whether the tag should be
+                active
+
+        Returns:
+            The response. See :py:meth:`send_command` for details.
+        """
+        if not name:
+
+            raise RequiredArgument(
+                "create_tag requires name argument")
+
+        if not resource_filter and not resource_ids:
+            raise RequiredArgument(
+                "create_tag requires resource_filter or resource_ids argument")
+
+        if not resource_type:
+            raise RequiredArgument(
+                "create_tag requires resource_type argument")
+
+        cmd = XmlCommand('create_tag')
+        cmd.add_element('name', name)
+        _xmlresources = cmd.add_element("resources")
+        if resource_filter is not None:
+            _xmlresources.set_attribute("filter", resource_filter)
+        for resource_id in resource_ids or []:
```

Comment:
```suggestion

        for resource_id in resource_ids or []:
```

Reply:
I think Github doesn't see this as addressed because i amended the original commit with it. Not sure why it worked for some changes.